receiver/kafkametricsreceiver/documentation.md

```diff
@@ -9,6 +9,15 @@ These are the metrics available for this scraper.
 | Name | Description | Unit | Type | Attributes |
 | ---- | ----------- | ---- | ---- | ---------- |
 | **kafka.brokers** | Number of brokers in the cluster. | {brokers} | Gauge(Int) | <ul> </ul> |
```

Comment:
Now we have a problem with metric names. According the the OTel spec metric name cannot be used as namespace for any other metrics. So we need to either pick another namespace for the added metrics or rename `kafka.brokers` through a deprecation process

Reply:
ah. I see this rule in the spec here:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-naming.md#attribute-naming.

I would prefer to change the added metrics namespace to avoid deprecation. Let me review the spec and other receivers and I can change the namespace.

Reply:
Let's discuss options here. I don't see a big problem in renaming `kafka.brokers` metric if we don't find a good alternative for the namespace

Reply:
I believe `kafka.brokers.count` would be the best option but happy to see other suggestions

Reply:
you are correct, `kafka.brokers.count` would be the best path forward per the spec:

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/README.md#use-count-instead-of-pluralization

Reply:
I am unsure of how to handle the rename/deprecation. This would be a breaking change and I don't see how to include that in the unreleased/*yaml file. 

I added a new attribute `kafka.brokers.count` that duplicates the `kafka.brokers` with a deprecation warning log, but am unsure if this is the correct way to handle this. Any guidance or links to spec docs would be helpful.

Reply:
Let's handle it in a separate PR and break it down into several releases, let's allow 2 versions for every step:
1. Introduce `kafka.brokers.count` as a new optional metric, add warning if `kafka.brokers` is enabled which is by default.
2. Enable `kafka.brokers.count` and disable `kafka.brokers` by default.
3. Remove `kafka.brokers` metric